sfs/array.py

```diff
@@ -422,39 +459,97 @@ def cube(N, spacing, center=[0, 0, 0], orientation=[1, 0, 0]):
     return ArrayData(positions, directions, weights)
 
 
-def sphere_load(fname, radius, center=[0, 0, 0]):
-    """Spherical secondary source distribution loaded from datafile.
+def sphere_load(file, radius, center=[0, 0, 0]):
+    """Load spherical secondary source distribution from file.
 
-    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 position, 1
-    weight) per secondary source located on the unit circle.
+    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 for the cartesian 
+    position vector, 1 for the integration weight) per secondary source located 
+    on the unit circle which is resized by the given radius and shifted to the
+    given center.
 
     Returns
     -------
     `ArrayData`
         Positions, orientations and weights of secondary sources.
 
+    Examples
+    --------
+    content of `example_array_6LS_3D.txt` 
+    ::
+        1 0 0 1
+        -1 0 0 1
+        0 1 0 1
+        0 -1 0 1
+        0 0 1 1
+        0 0 -1 1
+
+    corresponds to the `3-dimensional 6-point spherical 3-design
+    <http://neilsloane.com/sphdesigns/dim3/des.3.6.3.txt>`_.
+
+    .. plot::
+        :context: close-figs
+
+        x0, n0, a0 = sfs.array.sphere_load('../data/arrays/example_array_6LS_3D.txt', radius=2, center=[0, 0, 0])
```

Comment:
sfs/array.py:492:80: E501 line too long (113 > 79 characters)

In this case it's OK to exceed the line length in the source file because there are 8 spaces indentation which don't show up in the generated docs, but you should not exceed 87 characters in this case.

Reply:
changed to handling that is PEP8 ok